pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -11317,12 +11317,21 @@ Example: Crosslink to PEPTIDE: T [4]</comment>
   <data name="SmallMoleculeTransitionListReader_GetMoleculeTransitionGroup_Inconsistent_molecule_description" xml:space="preserve">
     <value>Inconsistent molecule description</value>
   </data>
+  <data name="DiaUmpireDdaConverter_Run_Starting_DIA_Umpire_conversion" xml:space="preserve">
```

Comment:
I am a little suspicious of all of the changes in this file because DIA-Umpire and DDASearchControl changes show up. You may be duplicating entries in this file. I had that problem with an auto-merge I did after Matt's commit.